packages/core/src/__integrationtests__/utils.ts

```diff
@@ -67,3 +90,111 @@ export const IsOfficialLicenseAuthority = CType.fromSchema({
   },
   type: 'object',
 })
+
+export async function initialTransfer(
+  faucet: KeyringPair,
+  addresses: Array<string>
+): Promise<void> {
+  await Promise.all(
+    addresses.map((address) =>
+      Balance.makeTransfer(address, ENDOWMENT).then((tx) =>
+        BlockchainUtils.signAndSubmitTx(tx, faucet, {
+          resolveOn: BlockchainUtils.IS_FINALIZED,
+          reSign: true,
+        }).catch((e) => console.log(e))
+      )
+    )
+  )
+}
+
+export async function getDidDeposit(didIdentifier: string): Promise<BN> {
```

Comment:
We can't depend on the DID package to return the deposit directly? Otherwise this is one more function we need to maintain and update every time something changes, and since it is inside the `__integrationtests__` folder not the very first thing that comes in mind. If it is a no-go to add a dev-dependency to the DID package, then you can ignore this comment.

Reply:
Yes, I have removed them from the PR.